docs/source/install.rst

```diff
@@ -102,17 +111,17 @@ Test Dependencies
         dask distributed cloudpickle
 
 
-UCX >= 1.11.1
+UCX >= 1.15.0
```

Comment:
You are right we don't officially support UCX < 1.15.0, nevertheless UCX-Py should still work with UCX >= 1.11.1, so the versions were left for the benefit of users who may still need older versions. In any case, I think it's fine to keep this bump in the docs too.

Reply:
Oh I didn't realize that was intentional. Happy to revert this if you want.

Reply:
No, it's ok to leave it like that. UCX < 1.15.0 is ancient anyway.

Reply:
Ha ok, thank you, sounds good.